dataflow/gpu-workers/landsat_view.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+
```

Comment:
I'm noticing that there's no corresponding test - happy to meet to talk about how to test this - I'm thinking you want an e2e test that
* pushes your container image to GCR
* calls whatever Dataflow thing is needed?
* looks for some kind of output

Happy to work with you on this - I think we're going to use a technique similar to what cloud run uses

Reply:
Yes, we were deferring the test for another PR just because they wanted the sample to merge quickly, but I plan to start working on the test soon.

But I would appreciate references on how to orchestrate all the steps.

Reply:
If products that you are using have client libraries, we definitely want to use them to create fixtures and do the necessary setup/teardown as needed. However, if not, we can consider doing something like we do with cloud run, which is to use the `subprocess` module to run `gcloud` as a workaround. Check out [this Cloud Run test](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/e2e_test.py) as an example - it uses the Pub/Sub library where it can, but uses the `subprocess` module for the pushing of the image to GCR and the deploying to Cloud Run steps